rpc_jobs/irr_role_test.yml

```diff
@@ -20,6 +20,9 @@
       - mitaka
       - liberty
       - kilo
+    scenario:
```

Comment:
``context`` was intended initially as ``scenario``, with ``series`` being used for the branch. I'd suggest changing this up so that ``series`` refers to the branch and rename ``context`` to ``scenario`` and using the scenario there.

Reply:
Series is the branch of the repo under test (rpc-maas), context in this case is the branch of rpco to test against. As only master is required, ceph could be an additional context, then we could add another axis later if required.